src/test/java/org/cactoos/io/ReaderOfTest.java

```diff
@@ -49,102 +48,101 @@
     @Rule
     public TemporaryFolder folder = new TemporaryFolder();
 
-    @Test(expected = NullPointerException.class)
-    public void readsNull() throws Exception {
-        MatcherAssert.assertThat(
-            new TextOf(new ReaderOf((Input) null)).asString(),
-            new IsEqual<>("")
-        );
-    }
-
     @Test
     public void readsEmpty() throws Exception {
-        MatcherAssert.assertThat(
-            new TextOf(new ReaderOf("")).asString(),
-            new IsEqual<>("")
-        );
+        new Assertion<>(
+            "must read empty string",
```

Comment:
@victornoel I think that would be better to create a constant with the `""` (empty string), WDYT?

Reply:
@paulodamaso you mean a variable in the test for the empty string? I can do that yes